static/templates/subscription_settings.hbs

```diff
@@ -2,46 +2,46 @@
     <div class="inner-box">
         <div class="alert stream_change_property_info alert-notification"></div>
 
-        {{#with sub}}
-        <div class="stream-header">
-            {{> subscription_privacy
-              invite_only=invite_only
-              is_web_public=is_web_public }}
-            <div class="stream-name">
-                <span class="sub-stream-name" title="{{name}}">{{name}}</span>
-            </div>
-            <div class="button-group">
-                <button id="open_stream_info_modal" class="button rounded small btn-warning" title="{{t 'Change stream info' }}">
-                    <i class="fa fa-pencil" aria-hidden="true"></i>
-                </button>
-                <div class="sub_unsub_button_wrapper inline-block">
-                    <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" {{#if should_display_subscription_button}}title="{{t 'Toggle subscription'}} (S)" {{else}}disabled="disabled"{{/if}}>
-                        {{#if subscribed }}{{t "Unsubscribe"}}{{else}}{{t "Subscribe"}}{{/if}}</button>
+        <div class="general_settings">
+            {{#with sub}}
+            <div class="stream-header">
+                {{> subscription_privacy
+                  invite_only=invite_only
+                  is_web_public=is_web_public }}
+                <div class="stream-name">
+                    <span class="sub-stream-name" title="{{name}}">{{name}}</span>
+                </div>
+                <div class="button-group">
+                    <button id="open_stream_info_modal" class="button rounded small btn-warning" title="{{t 'Change stream info' }}">
+                        <i class="fa fa-pencil" aria-hidden="true"></i>
+                    </button>
+                    <div class="sub_unsub_button_wrapper inline-block">
+                        <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" {{#if should_display_subscription_button}}title="{{t 'Toggle subscription'}} (S)" {{else}}disabled="disabled"{{/if}}>
+                            {{#if subscribed }}{{#tr}}Unsubscribe{{/tr}}{{else}}{{#tr}}Subscribe{{/tr}}{{/if}}</button>
+                    </div>
+                    <a href="{{preview_url}}" class="button small rounded" id="preview-stream-button" role="button" title="{{t 'View stream'}} (V)" {{#unless should_display_preview_button }}style="display: none"{{/unless}}><i class="fa fa-eye"></i></a>
+                    {{#if is_realm_admin}}
+                    <button class="button small rounded btn-danger deactivate" type="button" name="delete_button" title="{{t 'Delete stream'}}"> <i class="fa fa-trash-o" aria-hidden="true"></i></button>
```

Comment:
This s/Archive/Delete/` change here is wrong.

Reply:
Oops, fixed now.